ibis-server/tests/routers/ibis/test_clickhouse.py

```diff
@@ -122,7 +179,7 @@ def test_query(self, clickhouse: ClickHouseContainer):
         )
         assert response.status_code == 200
         result = response.json()
-        assert len(result["columns"]) == len(self.manifest["models"][0]["columns"])
+        assert len(result["columns"]) == 9
```

Comment:
Why use a constant instead of an object reference?

Reply:
Because the model has calculated fields now, the number of columns will be different. In dynamic query mode, the calculated fields won't be selected when select all.

Reply:
Oh, I see. Thanks.